lib/endpoints.js

```diff
@@ -101,6 +101,10 @@ module.exports = {
         resource: 'movie/:id/keywords',
         method: 'get'
       },
+      Recommend: {
```

Comment:
How about you rename this `Recommendations` to match the endpoint a little better